app/views/assignments/lti_settings.html.erb

```diff
@@ -0,0 +1,4 @@
+<div class='no-items-error'>
```

Comment:
In the controller, use `render layout: 'assignment_content'` to use that layout. That should have the `shared/flash_message` partial already included, so you won't need to include it here.

But instead, please supply a title for the view (you can see how other views do it, involves a `content_for` or something).